packages/core/js-client/src/clientPeer/__test__/client.spec.ts

```diff
@@ -178,7 +215,7 @@ describe("FluenceClient usage test suite", () => {
 
         expect(isConnected).toBeFalsy();
       });
-    });
+    }, 1000);
```

Comment:
move `1000` to constant, so it's possible to change for the whole test suite at once.